### PR TITLE
wrong error message on page product when user try to create product without title

### PR DESCRIPTION
**Pull request by Vlad1kent1**
## Code reviewers

@loqimean
@AndriyAndriyovuch

## Summary of issue

wrong error message on page product when user try to create product without title

## Summary of change

- change url link to the action create
- add commit
- button size correction

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

**Review by AndriyAndriyovuch — CHANGES_REQUESTED** [review]
Знімок екрану з 2023-07-13 14-51-45

And where's ukrainian locale?

Also you can fix buttons style (hint: look at @kos-nastiia previous PR)

File: app/views/account/products/new.html.slim [hunk 1]
```diff
@@ -1,5 +1,5 @@
 .container
-  = simple_form_for @product, method: :create, url: account_products_path, data: { controller: "price-form" } do |f|
+  = simple_form_for @product, url: { action: 'create'}, data: { controller: "price-form" } do |f|
```
Comment (AndriyAndriyovuch) [inline app/views/account/products/new.html.slim:2]:
```suggestion
  = simple_form_for @product, url: { action: 'create'}, data: { controller: "price-form" } do |f|
```

rubocop should show you error fot this, use it before push

File: config/locales/en/en.yml [hunk 2]
```diff
@@ -441,9 +441,9 @@ en:
         product:
           attributes:
             title:
-              blank: "The field 'Title' can't be blank"
-              too_long: "The field 'Title' is too long"
-              too_short: "The field 'Title' is too short."
+              # blank: "The field 'Title' can't be blank"
+              # too_long: "The field 'Title' is too long"
+              # too_short: "The field 'Title' is too short."
     form_errors:
```
[the remark this answers is not in the document]
Comment (Vlad1kent1) [reply config/locales/en/en.yml:447]: Line 579 has an error format consisting of %{attribute} and %{message}. In this case, %{attribute} is 'Title', and %{message} - these three lines or other messages below line 580. And if these three lines are not commented or deleted, then the result will be: 
Screenshot from 2023-07-14 01-12-19

File: app/views/account/products/new.html.slim [hunk 3]
```diff
@@ -1,5 +1,5 @@
...
```
Comment (loqimean) [inline app/views/account/products/new.html.slim:2]: який ще екшин, передавай норм урлу

Comment (Vlad1kent1) [reply app/views/account/products/new.html.slim]: виправив

**Review by loqimean — APPROVED** [review]

**Pull request merged**